.env.example

```diff
@@ -33,6 +33,12 @@ PHOTON_OSM_SEARCH_API_URL=PHOTONOSMSEARCHAPIURL
 #SSL_PRIVATE_KEY=/path/to/privkey.pem
 #SSL_CERT=/path/to/sslcert.pem
 
+# Set the 3 following variables to be able to route with the Transition public
+# API, with data from a specific Transition instance
+#TRANSITION_API_URL=http://ld2024.transition.city 
```

Comment:
We should put something like example.transition.city in the comment, not an actual site.